js.go

```diff
@@ -1127,6 +1162,121 @@ func (js *js) AddConsumer(stream string, cfg *ConsumerConfig) (*ConsumerInfo, er
 	return info.ConsumerInfo, nil
 }
 
+// JSApiConsumerDeleteResponse.
+type JSApiConsumerDeleteResponse struct {
+	APIResponse
+	Success bool `json:"success,omitempty"`
+}
+
+func (js *js) DeleteConsumer(stream, durable string) error {
```

Comment:
There is no "doc" of an exported function. Linters used to complain about this, not sure that is still the case. You could copy back what you have defined in the interface.